modules/nw-sriov-configuring-operator.adoc

```diff
@@ -59,32 +49,34 @@ Follow this procedure to disable it.
 * You must have installed the SR-IOV Operator.
 
 .Procedure
-. Update the default SriovOperatorConfig Custom Resource
+
+. To update the `default` SriovOperatorConfig CR, enter the following command:
```

Comment:
Suggest being clearer on what the user is doing instead of just saying "update". 

"To disable the Network Resources Injector, set the `enableInjector` field to `false`: